spikeinterface/sorters/runsorter.py

```diff
@@ -105,9 +105,23 @@ def run_sorter(
     )
 
     if docker_image or singularity_image:
+        if docker_image:
+            mode = "docker"
+            assert not singularity_image
+            if isinstance(docker_image, bool):
+                container_image = None
+            else:
+                container_image = docker_image
+        else:
+            mode = "singularity"
+            assert not docker_image
+            if isinstance(singularity_image, bool):
+                container_image = None
+            else:
+                container_image = singularity_image
```

Comment:
What do you think?

```suggestion
        assert not (docker_image and singularity_image)
        if docker_image:
            mode = "docker"
            container_image = None if docker_image is True else docker_image
        else:
            mode = "singularity"
            container_image = None if singularity_image is True else singularity_image
```

Reply:
I think that my version is easier to read in terms of logic :) in general, I don't like online conditions